liquidator/OneInchExchange.js

```diff
@@ -0,0 +1,135 @@
+const Web3 = require("web3");
```

Comment:
The current implementation of other scripts/modules or bot clients don't import `Web3` like this as they assume that: 
1) it's injected from truffle by running the `truffle exec` or `truffle test`
2) it's passed in from the base script.

In the context of this script the `web3` object would be passed in from the liquidator's `index.js` file where

Reply:
I understand. This was written to test the validity of swapping the tokens on 1inch. Outside of using truffle tests.

I normally use TDD, but because of how truffle test is structured (it compiles and migrates all the contracts beforehand), it would take ~5 before it actually got to testing. The iteration time was so slow and the error messages were so vague (`Invalid JSON RPC ""`), I decided to just hardcode it here and see if it passes.

Granted, this should have been removed :sweat:

Reply:
no problem, sounds good to me :)